test/functional/mempool_package_rbf.py

```diff
@@ -162,13 +162,13 @@ def test_package_rbf_additional_fees(self):
         self.log.info("Check replacement pays for incremental bandwidth")
         _, placeholder_txns3 = self.create_simple_package(coin)
         package_3_size = sum([tx.get_vsize() for tx in placeholder_txns3])
-        incremental_sats_required = Decimal(package_3_size) / COIN
-        incremental_sats_short = incremental_sats_required - Decimal("0.00000001")
+        incremental_sats_required = (Decimal(package_3_size * 0.1) / COIN).quantize(Decimal("0.00000001"))
+        incremental_sats_short = incremental_sats_required - Decimal("0.00000005")
         # Recreate the package with slightly higher fee once we know the size of the new package, but still short of required fee
         failure_package_hex3, failure_package_txns3 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_CHILD_FEE + incremental_sats_short)
         assert_equal(package_3_size, sum([tx.get_vsize() for tx in failure_package_txns3]))
         pkg_results3 = node.submitpackage(failure_package_hex3)
-        assert_equal(f"package RBF failed: insufficient anti-DoS fees, rejecting replacement {failure_package_txns3[1].txid_hex}, not enough additional fees to relay; {incremental_sats_short} < {incremental_sats_required}", pkg_results3["package_msg"])
+        assert_equal(f"package RBF failed: insufficient anti-DoS fees, rejecting replacement {failure_package_txns3[1].txid_hex}, not enough additional fees to relay; {incremental_sats_short:8f} < {incremental_sats_required:8f}", pkg_results3["package_msg"])
```

Comment:
I think you meant `:.8f` (8 decimal places) and not `:8f` ([width of 8](https://docs.python.org/3/library/string.html#format-specification-mini-language)) here?

Reply:
But also it seems that the error itself returned by bitcoind may not always have 8 decimal places?

Reply:
I haven't checked but i think as it stands the change is unnecessary. Doesn't warrant a retouch though, i just wish we ship this ASAP.

Reply:
This is the error, lmk if you had something else in mind?
```
AssertionError: not(package RBF failed: insufficient anti-DoS fees, rejecting replacement 4bc7d6a2a37b11c80cf4773012eaa50b8bb0f53327cbfccfec6b71705b829b49, not enough additional fees to relay; 1.6E-7 < 2.1E-7 == package RBF failed: insufficient anti-DoS fees, rejecting replacement 4bc7d6a2a37b11c80cf4773012eaa50b8bb0f53327cbfccfec6b71705b829b49, not enough additional fees to relay; 0.00000016 < 0.00000021)
```